src/AddressBook/ABAddressBook.cs

```diff
@@ -111,7 +134,14 @@ public class ABAddressBook : INativeObject, IDisposable, IEnumerable<ABRecord> {
 		[DllImport (Constants.AddressBookLibrary)]
 		internal extern static IntPtr ABAddressBookCreate ();
 
+#if !NET
 		[Deprecated (PlatformName.iOS, 6, 0, message : "Use the static Create method instead")]
+#else
+		[UnsupportedOSPlatform ("ios6.0")]
```

Comment:
The minimum supported OS version we support is not really helpful for `[UnsupportedOSPlatform]`. Our options are:

1. put a version-less `[UnsupportedOSPlatform ("ios")]` when the version is earlier than the minimum

The information is true, but it does not tell you when the API was removed or it it ever was supported on the platform. 

Not a big deal for a` 6.0`API  but if is was `10.0` (the minimum) then it can be good to know (since the replacement API are not always available right away).

It's also confusing/error-prone as adding `[NoiOS]` on bindings (which maps to `[UnsupportedOSPlatform ("ios")]`) would not generate the binding for that platform - which is not what happens for manual bindings.


2. put `[UnsupportedOSPlatform ("ios6.0")]` (not caring about the minimum OS version)

It's true and more complete. It you see code using this API you know it has not been updated recently :-)

> Do we need this?

Considering that we already have the information (when updating the attributes) and that sharpie can generate it (since it's in the headers) then I think it's cheap (to support) and useful to know (without requiring developers to check the headers/docs which are not always in sync).

It will also ease future updates of the attributes (when the minimum OS is bumped) since the `[UnsupportedOSPlatform]` won't ever need to be updated :-)